docs/_posts/2016-09-21-this-week-in-react-1.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "This Week In React #1"
+author: ericnakagawa
+---
+
+_This Week In React is a weekly publication highlighting interesting projects and active members from the React community. [Sign up to the weekly newsletter.](http://eepurl.com/cgOjbX)_
+
+![Projects aimed at beginning React developers.](/react/img/blog/thisweekinreact/beginner.png)
+
+####📓 A Visual Intro to Redux & Relay####
```

Comment:
Nit: this should be an h3, so only `###`. I think that should help make the rendered output a bit more readable.

formatting nit: space after the opening hashes (gets stripped, makes the markdown more readable). And remove the trailing ones.